src/app/shared/object-list/my-dspace-result-list-element/item-list-preview/item-list-preview.component.html

```diff
@@ -21,16 +21,16 @@ <h2 [innerHTML]="dsoTitle" [ngClass]="{'lead': true,'text-muted': !item.firstMet
                 [innerHTML]="item.firstMetadataValue('dc.publisher') + ', '"></span>
```

Comment:
This file doesn't include all the changes in #4737.  There's a missing change to line 15, where it should now use `item.hasMetadata()` instead of `item.firstMetadataValue`.  There's also a missing change to this dc.publisher line because the publisher field is no longer escaped. This should use: `item.firstMetadataValue('dc.publisher', undefined, true)`